PWGJE/Tasks/bjetTaggingML.cxx

```diff
@@ -0,0 +1,592 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
```

Comment:
why is there a separate task for ML here? why can the ML functionality not be added to the previous task

Reply:
This is different from the TreeCreator, this doesn't produce any trees or tables. I would like to keep the TreeCreator only for creating Trees so it stays clean and simple. And we apply the tagging in a different task. At some point later (after HP), this tagging will be done differently.